tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -180,4 +181,4 @@ steps:
       if [[ $? == 0 ]]; then
         echo "##vso[task.logissue type=error]Fluid internal dev versions shouldn't use caret dependencies"
         exit -1;
-      fi
\ No newline at end of file
+      fi
```

Comment:
looks like trailing blank line got dropped (I've seen VS Code do this to me too many times recently)

Reply:
That setting would help but without something in the build that enforces it it will get outdated again. Not everyone on the team uses vscode.

Reply:
To clarify, I am a fan of that setting and think we should add it; it's just an incomplete fix.

Reply:
Curious who ~is the traitor~ doesn't use VSCode 😄. I'll put out a separate PR to add these, I expect they'll still help since I'd guess non-VSCode use to be rare.